multimodal/src/autogluon/multimodal/utils/export.py

```diff
@@ -0,0 +1,235 @@
+import os
+import warnings
+from collections import defaultdict, namedtuple
+from typing import Dict, List, Optional, Union
+
+import pandas as pd
+import torch
+
+from ..constants import HF_TEXT, MMDET_IMAGE, TEXT, TIMM_IMAGE
+from ..models.fusion import AbstractMultimodalFusionModel
+from ..models.huggingface_text import HFAutoModelForTextPrediction
+from ..models.mmdet_image import MMDetAutoModelForObjectDetection
+from ..models.timm_image import TimmAutoModelForImagePrediction
+from .environment import compute_num_gpus, get_precision_context, infer_precision, move_to_device
+from .inference import process_batch
+from .onnx import get_onnx_input
+
+
+class ExportMixin:
```

Comment:
Will this Mixin also supports matcher?

Reply:
Yes, it would support matcher, and there is no API change.